plugins/src/main/java/opengrok/auth/plugin/LdapAttrPlugin.java

```diff
@@ -128,9 +128,8 @@ String getSessionAllowedAttrName() {
     public void fillSession(HttpServletRequest req, User user) {
         updateSession(req, false);
 
-        LdapUser ldapUser;
-        if ((ldapUser = (LdapUser) req.getSession().
-                getAttribute(LdapUserPlugin.getSessionAttrName(ldapUserInstance))) == null) {
+        LdapUser ldapUser = (LdapUser) req.getSession().                getAttribute(LdapUserPlugin.getSessionAttrName(ldapUserInstance));
```

Comment:
I missed the whitespaces :( 
```suggestion
        LdapUser ldapUser = (LdapUser) req.getSession().getAttribute(LdapUserPlugin.getSessionAttrName(ldapUserInstance));
```